### PR TITLE
Adds new flutter game Quokka

### LINKED ISSUE #313: Add new flutter game- "Quokka"
https://github.com/LinwoodDev/Quokka

### PR DESCRIPTION
**Pull request by arunimaChintu**
Closes #313

### CODE REVIEW

**Comment by arunimaChintu** [comment]
@ashitaprasad Please merge this.

**Comment by arunimaChintu** [comment]
@ashitaprasad Please merge this.

File: README.md [hunk 1]
```diff
@@ -146,9 +146,11 @@ All the projects added in this project are featured in [fluttergems.dev](flutter
 | 2048 | [Link](https://github.com/shubhexists/2048) |2048 game. |
 | Kunai Game | [Link](https://github.com/hzeyuan/flutter-flame-kunaiGame) | A simple Kunai game. |
 | Tetris | [Link](https://github.com/priyanshudutta04/Tetris) | A Tetris game. |
+| Quokka | [Link](https://github.com/LinwoodDev/Quokka) | A Quokka game. |
 | Minesweeper Game | [Link](https://github.com/recepsenoglu/minesweeper) | A Minesweeper game. |
 | Basketball Game | [Link](https://github.com/enoch-aik/basketball_game) | A Basketball game developed using Flame. |
 
+
```
Comment (ashitaprasad) [inline README.md:153]: Do not add empty new lines.

**Comment by arunimaChintu** [comment]
@ashitaprasad Removed the extra line... Please merge it now...

**Pull request merged**